esp/src/src-react/components/Metrics.tsx

```diff
@@ -181,7 +181,13 @@ export const Metrics: React.FunctionComponent<MetricsProps> = ({
                     (options.scopeTypes.indexOf(row.type) >= 0);
             }).map((row, idx) => {
                 row.__hpcc_id = row.name;
-                return [idx, row.type, row.name, ...options.properties.map(p => row[p] !== undefined ? row[p] : ""), row];
+                return [idx, row.type, row.name, ...options.properties.map(p => {
+                    return row[p] ||
+                        row.__groupedProps[p]?.Value ||
+                        row.__groupedProps[p]?.Max ||
+                        row.__groupedProps[p]?.Avg ||
+                        "";
```

Comment:
Change to `??` from `||`.  Will need:  https://github.com/hpcc-systems/Visualization/pull/4162

Reply:
Can you clarify.  Is this something you plan to change in the future?  It looks like you have merged that changed now.
Should I merge this, or wait until you modify?